apps/arweave/src/ar.erl

```diff
@@ -311,7 +311,8 @@ show_help() ->
 			{"is_pool_client", "Configure the node as a pool client. The node may be an "
 					"exit peer in the coordinated mining setup or a standalone node."},
 			{"pool_api_key", "API key for the requests to the pool."},
-			{"pool_server_address", "The pool address"}
+			{"pool_server_address", "The pool address"},
+			{"pool_worker_name", "The pool worker name"}
```

Comment:
is `pool_worker` the same as `pool_client`? i.e. Above we have a `is_pool_client` config. if `pool_worker` and `pool_client` are the same, can we call this `pool_client_name`?

Reply:
We can call it pool_client_name, but it will be more confusing to miners from other cryptocurrencies because anybody else it's calling either `worker` or `worker name`

Reply:
Got it. I do think we should be consistent within our own code first and then consistent across blockchains second. 

Do you want to update all 'pool client" instances to be "pool worker"? Sounds like that might be the best approach - although it'll be a pain to handle compatibility. 

Otherwise as a quick fix is suggest we continue using pool client until we can make a consistent update across the codebase. Having two names will just make things harder to maintain (and will be confusing to miners as well when they are configuring their miner)

Reply:
we have already intersection in names
worker is kinda reserved for worker inside mining process
client is for connecting pool
inside client there is just label what "pool label" called "worker" should be displayed on pool

Reply:
All I'm saying is that right her in this section of code we hvae 3 different names:
client
server
worker

a user seeing that will assume they refer to 3 different roles. If some of them refer to the same role we should name them the same thing. Otherwise we'll get questions about whether or not they have to set `is_pool_client` if they want to set their `pool_worker_name`

I don't have a strong opinion about client vs. worker. but I do have a strong opinion that we should be consistent - particularly within a given functinal role.

I don't get what the problem is here? we literallly call it "pool_client" 2 lines earlier, doesn't it makes sense that it will be confusing if we cal it "pool_worker" here? 

Again I"m fine if we want to refer to this entity as a "pool_worker" everywhere, but I *really* don't want to introduce inconsistency that we don't have to. It just makes the code harder to understand, use, and maintain for no good reason.

Reply:
`pool_client` is mode of our arweave node
`pool_worker_name` is optional label on pool, that's why I added name
We can explicitly say in description that it's optional

Reply:
Put another way. 

We have is_pool_client and is_pool_server implying those are the two roles you can configure your miner as.

if we introduce `pool_worker_name` the implication is that that is a 3rd role and we would need `is_pool_worker` - or the implication is that it independent of the two roles and so you could set `pool_worker_name` even if you set `is_pool_server`

it just gets needlessly confusing since the only time you set the name is if you ar a `pool_client` so it makes sense that it woudl be `pool_client_name`

Reply:
pool_client_worker_name?

Reply:
haha, I honestly do not understand what the difference is between a worker and a client. To me they are synonyms. If you think they are different thing that need to be referred differentl: okay, I can go along with that. But I personally would like to see it in the config flag help  - some explanation for what a "worker" is and why it is distinct from a "client"

Reply:
Added more verbose description

Reply:
Deal 🤝 

Looks good. I added a suggested space so the words don't run together.